auth0/auth0-deploy-cli
# Fix proxying (in Node 10) and the debug flag.

## ✏️ Changes

- Fix the `--proxy-url` switch to work in Deploy CLI > 5 and >= Node 10.
- Also change the --debug switch so it no longer requires a "true/false" to work. (It is still backwards compatible if one is provided).

## 🔗 References

https://auth0team.atlassian.net/browse/ESD-7475

## 🎯 Testing

- Run a local proxy: i used `docker run -it -p 8899:8899 --rm abhinavsingh/proxy.py:latest`
- On Node 10 or higher set `--proxy_url http://localhost:8899`
- Ensure the traffic flows through the proxy.

🚫 This change has unit test coverage
🚫 This change has integration test coverage
🚫 This change has been tested for performance
